source/features/pr-branches.tsx

```diff
@@ -82,35 +86,35 @@ function buildQuery(numbers: number[]) {
 function createLink(ref: RepositoryReference) {
 	return (
 		<span
-			class="commit-ref css-truncate user-select-contain mb-n1"
-			style={(ref.branchExists ? {} : {'text-decoration': 'line-through'})}>
+			className="commit-ref css-truncate user-select-contain mb-n1"
+			style={(ref.branchExists ? {} : {textDecoration: 'line-through'})}>
 			{
 				ref.url ?
 					<a title={(ref.branchExists ? ref.label : 'Deleted')} href={ref.url}>
 						{ref.label}
 					</a> :
-					<span class="unknown-repo">unknown repository</span>
+					<span className="unknown-repo">unknown repository</span>
 			}
 		</span>
 	);
 }
 
-async function init() {
+async function init(): Promise<boolean|void> {
 	const elements = select.all('.js-issue-row');
 	if (elements.length === 0) {
 		return false;
 	}
 
 	const {ownerName} = getOwnerAndRepo();
-	const query = buildQuery(elements.map(pr => pr.id));
+	const query = buildQuery(elements.map(pr => parseInt(pr.id, 10)));
```

Comment:
`parseInt(pr.id, 10)`his could probably just be `Number(pr.id)`.

Reply:
Yeah I always prefer `Number`

Reply:
done